configure

```diff
@@ -5570,9 +5570,9 @@ if test $usingTcl ; then
 
     *-freebsd*)
       # Not available on all versions:  check for include file.
-      SHLIB_CFLAGS="-fpic"
+      SHLIB_CFLAGS="-fPIC"
```

Comment:
If qflow is to support multiple hosts, then I believe we need a smoke-test Dockerfile to test each of those hosts. The above just seems like it was never tested?

Reply:
AFAIK when I was compiling on my FreeBSD machine, it wouldn't work if I didn't do this. A more proper testing system is probably needed at some point.